Packs/CommonTypes/IndicatorTypes/reputation-cve.json

```diff
@@ -3,164 +3,69 @@
+                ]
             }
-          ]
+        },
+        "cvssvector": {
+            "simple": "CVE.CVSS.Vector",
+            "complex": null
+        },
+        "cvssversion": {
+            "simple": "CVE.CVSS.Version",
+            "complex": null
+        },
+        "description": {
+            "simple": "CVE.Description",
+            "complex": null
+        },
+        "stixid": {
+            "simple": "CVE.STIXID",
+            "complex": null
         }
-      },
-      "cvss": {
-        "simple": "",
-        "complex": {
-          "root": "CVE",
-          "filters": [],
-          "accessor": "CVSS",
-          "transformers": [
-            {
-              "operator": "uniq",
-              "args": {}
-            }
-          ]
-        }
-      },
-      "indicatoridentification": {
-        "simple": "",
-        "complex": {
-          "root": "CVE",
-          "filters": [],
-          "accessor": "ID",
-          "transformers": [
-            {
-              "operator": "toUpperCase",
-              "args": {}
-            },
-            {
-              "operator": "uniq",
-              "args": {}
-            }
-          ]
-        }
-      },
-      "cvemodified": {
-        "simple": "",
-        "complex": {
-          "root": "CVE",
-          "filters": [],
-          "accessor": "Modified",
-          "transformers": [
-            {
-              "operator": "uniq",
-              "args": {}
-            }
-          ]
-        }
-      },
-      "published": {
-        "simple": "",
-        "complex": {
-          "root": "CVE",
-          "filters": [],
-          "accessor": "Published",
-          "transformers": [
-            {
-              "operator": "uniq",
-              "args": {}
-            }
-          ]
-        }
-      },
-      "associations": {
-        "simple": "",
-        "complex": {
-          "root": "CVE",
-          "filters": [],
-          "accessor": "Associations",
-          "transformers": [{
-            "operator": "uniq",
-            "args": {}
-          }]
-        }
-      },
-      "reportedby": {
-        "simple": "",
-        "complex": {
-          "root": "CVE",
-          "filters": [],
-          "accessor": "ReportedBy",
-          "transformers": [{
-            "operator": "uniq",
-            "args": {}
-          }]
-        }
-      },
-      "threattypes": {
-        "simple": "",
-        "complex": {
-          "root": "CVE",
-          "filters": [],
-          "accessor": "ThreatTypes",
-          "transformers": [{
-            "operator": "uniq",
-            "args": {}
-          }]
-        }
-      },
-      "trafficlightprotocol": {
-        "simple": "",
-        "complex": {
-          "root": "CVE",
-          "filters": [],
-          "accessor": "TrafficLightProtocol",
-          "transformers": [{
-            "operator": "uniq",
-            "args": {}
-          }]
-        }
-      },
-      "tags": {
```

Comment:
we dont want to keep it for the CVE indicator?

Reply:
Yarden, you need full comments, I can't follow 😄

Reply:
It isnt deleted, just moved up